modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -491,6 +495,10 @@ def test_loc_setting_single_categorical_column():
     df_equals(modin_df, pandas_df)
 
 
+@pytest.mark.skipif(
+    InitializeWithSmallQueryCompilers.get(),
+    reason="SmallQueryCompiler does not currently support IO functions.",
```

Comment:
doesn't this "not supporting" make current PR a draft until it really does support I/O?